package.json

```diff
@@ -128,5 +128,12 @@
     "test-cov": "nyc --require=flow-remove-types/register --reporter=text-summary --reporter=lcov --cache run-s test-unit test-render test-query",
     "prepublish": "in-publish && run-s build-dev build-min || not-in-publish",
     "postinstall": "cd src/style-spec && yarn || npm install"
-  }
+  },
+  "files": [
+    "src/",
+    "dist/",
+    "plugins/",
+    "LICENSE.txt",
+    "ARCHITECTURE.md"
```

Comment:
I don't think we'd want to include `ARCHITECTURE.md` in the npm package. We should include `README.md` and `CHANGELOG.md` though. 

Also the `plugins/` folder isn't needed (it actually shouldn't even be in `master` anymore)